src/lib/commands/serverlessFunction.ts

```diff
@@ -0,0 +1,19 @@
+import * as vscode from 'vscode';
+import { convertFolderToServerlessFunction } from '../fileHelpers';
+import { COMMANDS } from '../constants';
+
+export const registerCommands = (context: vscode.ExtensionContext) => {
+  context.subscriptions.push(
+    vscode.commands.registerCommand(
+      COMMANDS.CREATE_SERVERLESS_FUNCTION_FOLDER,
+      async (clickContext) => {
```

Comment:
Does this need to be async?

Reply:
Looks like no! @miketalley had this on his original module create code and I blindly copied it 😶 removed!